src/status_im/utils/styles.cljs

```diff
@@ -0,0 +1,3 @@
+(ns status-im.utils.styles
+  (:require-macros status-im.utils.fx)
```

Comment:
@yenda, shouldn't this be `status-im.utils.styles`?

Reply:
omg you are right

Reply:
did you try with styles there? does it work?

Reply:
Tried and for some reason I'm getting this when run `make release-desktop`
```
WARNING: Use of undeclared Var status-im.utils.platform/os at line 7 /Users/vkjr/work/projects/github/status-im2/status-react/src/status_im/ui/screens/browser/styles.cljs

Error encountered performing task 'cljsbuild' with profile(s): 'prod'
Subprocess failed
```

Reply:
I'll rebase PR tomorrow and will check again. Too sleepy now :)